src/30_Advanced/Tab_Panels_with_amp-selector.html

```diff
@@ -0,0 +1,112 @@
+<!--
+  #### Introduction
+
+  This is a implementation of tab panels that uses [`amp-selector`](https://www.ampproject.org/docs/reference/components/dynamic/amp-selector). Tabs and tab panels are positioned via `flexbox`
+  and `amp-selector` uses the `selected` attribute to show/hide the correct tab panel and style the selected tab.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+
+<head>
+    <meta charset="utf-8">
+    <link rel="canonical" href="<%host%>/advanced/tab_panels_with_amp-selector/">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- #### Setup -->
+    <!-- Import `amp-selector` in the header -->
+    <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+    <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
+    <!-- You will also need a bit of extra styling for actual functionality -->
+    <style amp-custom="">       
+        .ampTabContainer {
+            display: flex;
+            flex-wrap: wrap;
+        }
+        
+        .tabButton[selected] {
+            outline: none;
+            background: #ccc;
+        }
+        
+        .tabButton {
+            list-style: none;
+            flex-grow: 1;
+            text-align: center;
+            cursor: pointer;
+        }
+        
+        .tabContent {
+            display: none;
+            width: 100%;
+            order: 1; /* must be greater than the order of the tab buttons to flex to the next line */
+            border: 1px solid #ccc;
+        }
+        
+        .tabButton[selected]+.tabContent {
+            display: block;
+        }
+
+        .itemCustom {
+            border: 1px solid #000;
+            height: 280px;
+            width: 380px;
+            margin: 10px;
+            text-align: center;
+            padding-top: 140px;
```

Comment:
This is optional styling to quickly center text in the custom item in the carousel. If you think it's not needed I can remove it